packages/shared/lib/contexts/onboarding/stores/shimmer-claiming-transactions.store.ts

```diff
@@ -0,0 +1,44 @@
+import { get, Writable } from 'svelte/store'
+
+import { InvalidTransactionIdError } from '@core/wallet'
+import { persistent } from '@lib/helpers'
+
+import { IShimmerClaimingTransactionStore } from '../interfaces'
+
+import { onboardingProfile } from './onboarding-profile.store'
```

Comment:
```suggestion
import { get, Writable } from 'svelte/store'
import { InvalidTransactionIdError } from '@core/wallet'
import { persistent } from '@lib/helpers'
import { IShimmerClaimingTransactionStore } from '../interfaces'
import { onboardingProfile } from './onboarding-profile.store'
```

Reply:
I've been writing imports like this for a while, with a line separating the different levels (svelte -> iota/wallet -> external lib modules -> internal lib module). We should look at specifying import rules with ESLint to ultimately get the most consistent setup.

Reply:
Created a [task](https://github.com/iotaledger/firefly/issues/4339) ~